composables/transaction/transactionSend.ts

```diff
@@ -28,12 +32,16 @@ function checkTsxSend(item: ActionSend) {
 }
 
 function execSendRmrk(item: ActionSend, api, executeTransaction) {
-  const version = item.urlPrefix === 'rmrk' ? '1.0.0' : '2.0.0'
+  const interaction =
+    item.urlPrefix === 'rmrk'
+      ? createInteraction(Interaction.SEND, item.nftId, item.address)
+      : createNewInteraction({
+          action: NewInteraction.SEND,
+          payload: { id: item.nftId, recipient: item.address },
+        })
```

Comment:
this check repeats.
possible to extract it into a function?

Reply:
Added composable util in [a3f5058](https://github.com/kodadot/nft-gallery/pull/5631/commits/a3f5058de95e1023f354a355f854d3a30c30f0b3)

Can be done in other PR